crates/ser/src/types/sync.rs

```diff
@@ -0,0 +1,68 @@
+//! Sync types for the optimism rollup node.
+
+use alloy::{
+    primitives::{BlockNumber, B256},
+    rpc::types::BlockId,
+};
+
+/// The block reference for an L2 block.
+///
+/// See: <https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/id.go#L33>
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct L2BlockRef {
+    /// The block hash.
+    pub hash: B256,
+    /// The block number.
+    pub number: BlockNumber,
+    /// The parent hash.
+    pub parent_hash: B256,
+    /// The timestamp.
+    pub timestamp: u64,
+    /// The L1 origin.
+    #[serde(rename = "l1Origin")]
+    pub l1_origin: BlockId,
+    /// The sequence number.
+    pub sequence_number: u64,
+}
+
+/// The block reference for an L1 block.
+///
+/// See: <https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/id.go#L52>
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct L1BlockRef {
+    /// The block hash.
+    pub hash: B256,
+    /// The block number.
+    pub number: BlockNumber,
+    /// The parent hash.
+    pub parent_hash: B256,
+    /// The timestamp.
+    pub timestamp: u64,
+}
+
+/// The [`SyncStatus`][ss] of an Optimism Rollup Node.
+///
+/// [ss]: https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/sync_status.go#L5
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct SyncStatus {
```

Comment:
before we introduce these here,

could we instead move all of these 

https://github.com/paradigmxyz/reth/blob/c788b6a585bbbd7ed92aabc42cb612a6a804daff/crates/optimism/rpc/src/api.rs#L33-L33

to op-alloy instead and import?

but could also do this in a followup

Reply:
Looks like someone saw this and already put up a pr :) https://github.com/alloy-rs/op-alloy/pull/35

Reply:
Happy to wait until that's in and re-use those types.